ibis/sql/alchemy.py

```diff
@@ -881,7 +967,18 @@ def _fetch(self, cursor):
             columns=cursor.proxy.keys(),
             coerce_float=True,
         )
-        return self.schema().apply_to(df)
+        df = self.schema().apply_to(df)
+        if GEO:
+            geom_col = None
+            for name, dtype in self.schema().items():
+                if isinstance(dtype, dt.GeoSpatial):
+                    geom_col = geom_col or name
+                    df[name] = df.apply(
```

Comment:
Do we need to use `apply` here? It looks like you're calling `shape.to_shape` on a `Series`, which would only require pulling out `df[name]`. Am I missing something here?

Reply:
I think I'm using `shape.to_shape` on a particular field, row-wise, rather than on the whole `Series` here.

Reply:
Ok, I can never remember what the `axis` argument means in pandas :p

Reply:
Yeah, I have a piece of paper pinned up at my desk with "AXIS=1" written on it, since it's usually what I want and I always for et it.